tests-extended/makefile

```diff
@@ -1,6 +1,8 @@
 #!/usr/bin/make -f
 
 LDFLAGS += -lboost_system
+# XXX: can be removed when we delete deprecated DNS logger methods
```

Comment:
jestli tu neni XXX uz nekde pouzivane, radsi bych videl TODO

Reply:
As Adam said, XXX is a pattern representing something that ought to be fixed, but cannot be changed right now. So OK from me.